includes/User.php

```diff
@@ -2131,7 +2138,9 @@ public function setInternalPassword( $str ) {
 		$this->load();
 		$this->setToken();
 
-		if( $str === null ) {
+		// @deprecated
+		// @todo remove local storing of password
+		if ( $str === null ) {
 			// Save an invalid hash...
 			$this->mPassword = '';
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2068)